opentelemetry-sdk/tests/performance/benchmarks/metrics/test_benchmark_metrics_histogram,.py

```diff
@@ -0,0 +1,126 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import random
+
+import pytest
+
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import InMemoryMetricReader
+from opentelemetry.sdk.metrics.view import (
+    ExplicitBucketHistogramAggregation,
+    View,
+)
+
+MAX_BOUND_VALUE = 10000
+
+
+def _generate_bounds(bound_count):
+    bounds = []
+    for i in range(bound_count):
+        bounds.append(i * MAX_BOUND_VALUE / bound_count)
+    return bounds
+
+
+hist_view_10 = View(
+    instrument_name="test_histogram_10_bound",
+    aggregation=ExplicitBucketHistogramAggregation(_generate_bounds(10)),
+)
+hist_view_49 = View(
+    instrument_name="test_histogram_49_bound",
+    aggregation=ExplicitBucketHistogramAggregation(_generate_bounds(49)),
+)
+hist_view_50 = View(
+    instrument_name="test_histogram_50_bound",
+    aggregation=ExplicitBucketHistogramAggregation(_generate_bounds(50)),
+)
+hist_view_1000 = View(
+    instrument_name="test_histogram_1000_bound",
+    aggregation=ExplicitBucketHistogramAggregation(_generate_bounds(1000)),
+)
+reader = InMemoryMetricReader()
+provider = MeterProvider(
+    metric_readers=[reader],
+    views=[
+        hist_view_10,
+        hist_view_49,
+        hist_view_50,
+        hist_view_1000,
+    ],
+)
+meter = provider.get_meter("sdk_meter_provider")
+hist = meter.create_histogram("test_histogram_default")
+hist10 = meter.create_histogram("test_histogram_10_bound")
+hist49 = meter.create_histogram("test_histogram_49_bound")
+hist50 = meter.create_histogram("test_histogram_50_bound")
+hist1000 = meter.create_histogram("test_histogram_1000_bound")
```

Comment:
Would it be simpler to just inject a new separate MeterProvider for each test with just a single instrument and we wouldn't need views or multiple instruments at all

Reply:
How do we test the custom boundaries cases without views? I don't believe there is a way to configure boundaries of the aggregation?

Reply:
You're right you still need one view, but wouldn't need separate instruments for each test case, just a single view and histogram per each.